common/data_format.hpp

```diff
@@ -169,14 +160,19 @@ struct rendered_frame : public switchboard::event {
 };
 
 struct hologram_input : public switchboard::event {
-    ullong seq;
+    uint seq;
 
     hologram_input() { }
 
-    hologram_input(ullong seq_)
+    hologram_input(uint seq_)
         : seq{seq_} { }
 };
 
+/* I use "accel" instead of "3-vector" as a datatype, because
+this checks that you meant to use an acceleration in a certain
+place. */
+struct accel { };
```

Comment:
where is this used?

Reply:
Please check if it's used somewhere. If not, remove it.